tox.ini

```diff
@@ -40,7 +40,7 @@ max_line_length = 88
 basepython = python3.7
 deps = black
 commands =
-    black . --target-version py27 --exclude setup.py
+    black . --target-version py27 --exclude (\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|\.svn|_build|buck-out|build|dist)/|/setup.py
```

Comment:
.hg, .venv, _build, buck-out and .eggs are not folders that should usually exist in here, no?
Also setup.py has a leading /, which means it wouldn't match the setup.py in the current folder

Reply:
same  for .nox